src/platforms/javascript/common/troubleshooting/index.mdx

```diff
@@ -81,7 +81,7 @@ A tunnel is an HTTP endpoint that acts as a proxy between Sentry and your applic
 
 Starting with version `6.7.0` of the JavaScript SDK, you can use the `tunnel` option to tell the SDK to deliver events to the configured URL, instead of using the DSN. This allows the SDK to remove `sentry_key` from the query parameters, which is one of the main reasons ad-blockers prevent sending events in the first place. This option also stops the SDK from sending preflight requests, which was one of the requirements that necessitated sending the `sentry_key` in the query parameters.
 
-To enable the `tunnel` option, provide either a relative or an absolute URL in your `Sentry.init` call:
+To enable the `tunnel` option, provide either a relative or an absolute URL in your `Sentry.init` call. A URL is relative to the current origin and this is the form that we recommend. Using relative url will not trigger a pre-flight CORS request and no events will be ever blocked as ad-blocker will not treat it as a 3rd party request.
```

Comment:
Tried to make this a bit clearer, but please make sure I haven't changed the meaning.

```suggestion
To enable the `tunnel` option, provide either a relative or an absolute URL in your `Sentry.init` call. When you use a relative URL, it's relative to the current origin, and this is the form that we recommend. Using a relative URL will not trigger a preflight CORS request, so no events will be blocked, because the ad-blocker will not treat these events as third-party requests.
```